doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/project-deployment-options-ref.adoc

```diff
@@ -0,0 +1,57 @@
+[id='project-deployment-options-ref_{context}']
+= Project deployment options with {PRODUCT}
+
+After you develop, test, and build your {PRODUCT} project, you can deploy the project to begin using the business assets you have created. You can deploy a {PRODUCT} project to a configured {KIE_SERVER}, to an embedded Java application, or into a {OPENSHIFT} environment for an enhanced containerized implementation.
+
+The following options are the main methods for {PRODUCT} project deployment:
+
+.Project deployment options
+[cols="25%,50%,25%", options="header"]
+|===
+|Deployment option
+|Description
+|Documentation
+
+ifdef::DM,PAM[]
+|Deployment to an OpenShift environment
+|{OPENSHIFT} brings together Docker and Kubernetes and enables you to create and manage containers. You can install both {CENTRAL} and {KIE_SERVER} on OpenShift. {PRODUCT} provides templates that you can use to deploy a {PRODUCT} authoring environment, managed server environment, immutable server environment, or trial environment on OpenShift. With OpenShift, components of {PRODUCT} are deployed as separate OpenShift pods. You can scale each of the pods up and down individually, providing as few or as many containers as necessary for a particular component. You can use standard OpenShift methods to manage the pods and balance the load.
```

Comment:
Suggestions for rewrite:

s/{OPENSHIFT} brings together Docker and Kubernetes and enables you to create and manage containers. You can install both {CENTRAL} and {KIE_SERVER} on OpenShift./{OPENSHIFT} integrates Docker with Kubernetes. You can create and manage containers, and install both {CENTRAL} and {KIE_SERVER} on OpenShift.

s/{PRODUCT} provides templates that you can use to deploy a {PRODUCT} authoring environment, managed server environment, immutable server environment, or trial environment on OpenShift./You can use {PRODUCT} templates to deploy a {PRODUCT} authoring environment, managed server environment, immutable server environment, or trial environment on OpenShift.

Reply:
These are examples where I disagree. The first (although I did change "brings together" to "combines") is more awkward than it's worth when de-anthropomorphized :) and the second I says something slightly different when rewritten accordingly. I want the second focusing on the product and what it provides, not just what you can do with it. There's a subtle but important diff, imo.

Reply:
You are quite right. I reread these while considering your points, along with the previous comment about "X enables you to ...", and your original phrasing conveys this better.